cql3/attributes.cc

```diff
@@ -57,16 +56,13 @@ int64_t attributes::get_timestamp(int64_t now, const query_options& options) {
 }
 
 int32_t attributes::get_time_to_live(const query_options& options) {
-    if (!_time_to_live.has_value())
+    if (!_time_to_live.has_value() || _time_to_live_unset_guard.should_skip_operation(options))
```

Comment:
I think the has_value() test is wrong (see https://github.com/scylladb/scylladb/issues/12243) but this is preexisting code and no need to fix it in this patch.